README.md

```diff
@@ -36,15 +36,27 @@ Der Server bietet sowohl für die App als auch für den Bot die notwendigen API
 Der Source-Code und Dokumentation des Servers ist in diesem Repo zu finden.
 
 #### Install and Start
-Voraussetzung ist ein installiertes python 3.8 package
-- `sudo ./manage.sh install` um den server zu installieren
-  (sudo only necessary to install graphviz to display story graph)
-- `./manage.sh start` um den server zu starten
-- für weitere befehle `./manage.sh help` benutzen
+Voraussetzung ist ein Linux System mit installiertem python 3.8
+1. `./manage.sh install` um den server zu installieren
+1. `./manage.sh start` um den server zu starten
+1. für weitere befehle `./manage.sh help` benutzen
+1. siehe Bot repo für Installation und Start des Bots
+1. nach Start von Server und Bot kann die App und damit das Spiel gestartet werden
 
 #### Architektur-Überblick
 Der Server wird durch eine Flask App (`/app`) implementiert. Diese verwaltet verschiedene API-Endpunkte (`/app/routes`). Außerdem wird eine Datenbank verwaltet. Deren ORM wird in `/app/models` implementiert. Der Story-Inhalt und der Story-verwaltende Code liegt unter `/app/story`.  
 
+#### Nutzung ohne App
+Unsere App ist sehr datenhungrig. Um das Spiel zu testen, ohne die App zu installieren, haben wir ein Postman package erstellt, welches die App immitiert mit dem sich das Spiel durchspielen lässt (die Spielerfahrung leidet darunter jedoch deutlich). Die Nutzung ist wie folgt:
+1. `detective-game-no-app-walkthrough.json` mit Postman importieren
```

Comment:
Request `mocked_contacts` lead to the player being called `Tim` and the analyst called `Tim Garrels` for me.
This seems like a mistake that should be fixed by sending more contact data. (maybe a server issue selecting the same contact twice?)

Reply:
I sent 4 mocked Contacts, namely us 4 creators. So your contact Tim is set for the analyst, but your playername is your name from telegram. So if the correctors play this, there should be no problem.
However, more contacts would still be nice, because there is the possibility for choosing the same contact for analyst and suspect, because we only have 4 contacts.
I will add some Gameprog participants